sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/samples/DashboardSampleFilterLayout.java

```diff
@@ -0,0 +1,65 @@
+package de.symeda.sormas.ui.dashboard.samples;
+
+import com.vaadin.icons.VaadinIcons;
+import com.vaadin.server.Sizeable.Unit;
+import com.vaadin.shared.ui.ContentMode;
+import com.vaadin.ui.Alignment;
+import com.vaadin.ui.Label;
+import com.vaadin.v7.ui.ComboBox;
+
+import de.symeda.sormas.api.Disease;
+import de.symeda.sormas.api.FacadeProvider;
+import de.symeda.sormas.api.i18n.I18nProperties;
+import de.symeda.sormas.api.i18n.Strings;
+import de.symeda.sormas.ui.dashboard.AbstractDashboardView;
+import de.symeda.sormas.ui.dashboard.DashboardDataProvider;
+import de.symeda.sormas.ui.dashboard.DashboardType;
+import de.symeda.sormas.ui.dashboard.components.DashboardFilterLayout;
+import de.symeda.sormas.ui.utils.CssStyles;
+
+public class DashboardSampleFilterLayout extends DashboardFilterLayout{
+	private ComboBox diseaseFilter;
+
+	public DashboardSampleFilterLayout(AbstractDashboardView dashboardView, DashboardDataProvider dashboardDataProvider) {
+		super(dashboardView, dashboardDataProvider, null);
+		setStyleName(CssStyles.D_FLEX, true);
+		setStyleName(CssStyles.FLEX_WRAP, true);
+	}
+
+	@Override
+	public void populateLayout() {
+		createDateFilters();
+		createInfoLabel();
+		createRegionAndDistrictFilter();
+		if (dashboardDataProvider.getDashboardType() == DashboardType.SAMPLES) {
```

Comment:
This if clause is probably pretty redundant since this class should only be used for the sample dashboard in the first place